src/utils/position/position.ts

```diff
@@ -20,7 +20,7 @@ export const positionVariants = (
     width: number,
     overlayHeight: number,
     overlayWidth: number,
-    spacing: number = 0
+    spacing = 0
```

Comment:
was this removed for redundancy sake?

Reply:
This is a lint rule I enabled. The tl;dr: is do not allow people to specify a type redundantly when it can be inferred from the instantiation.